Scripts/Python/ki/xKIChat.py

```diff
@@ -44,6 +44,7 @@
 import re
 import time
 import random
+from inspect import signature
```

Comment:
I have a slight preference for importing just `inspect` and calling `inspect.signature()` for clarity.